compiler/src/test-integration/java/io/neow3j/compiler/CryptoLibIntegrationTest.java

```diff
@@ -53,12 +53,12 @@ public void verifyWithECDsa() throws IOException {
         String signature =
                 "a30ded6e19be5573a6f6a5ff37c35d4ae76ff35ab4bee03b5b5bfbbef371f812ff70b5b480462807948a2ffb24dd8771484d9ca5a90333f9e6db69a6c8802a63";
         NeoInvokeFunction response = ct.callInvokeFunction(testName, byteArray(message),
-                publicKey(pubKey), byteArray(signature), integer(NamedCurve.SECP256R1));
+                publicKey(pubKey), byteArray(signature), integer(NamedCurve.secp256r1));
```

Comment:
This should be `Secp256r1` now. The CI did not catch that, since the integration test files are not included in the build.

```suggestion
                publicKey(pubKey), byteArray(signature), integer(NamedCurve.Secp256r1));
```